frontend/packages/dev-console/src/components/topology/__tests__/topology-page.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { StatusBox, LoadingBox } from '@console/internal/components/utils';
+import Topology from '../Topology';
+import { TopologyPage, RenderTopology } from '../TopologyPage';
+import ProjectsExistWrapper from '../../ProjectsExistWrapper';
+import { topologyData } from './topology-test-data';
+
+type TopologyPageProps = React.ComponentProps<typeof TopologyPage>;
+type RenderTopologyProps = React.ComponentProps<typeof RenderTopology>;
+
+let topologyProps: TopologyPageProps;
+let renderTopologyProps: RenderTopologyProps;
+
+describe('Topology page tests', () => {
```

Comment:
Add test cases to verify topology list view & project with no workloads.

Reply:
I already added this use case. see here https://github.com/openshift/console/pull/3829/files#diff-3ff93205d605f09dfbf10bf6adf7dfcbR50